assets/css/admin/componets/_footer.scss

```diff
@@ -0,0 +1,4 @@
+.footer {
+    padding: 165px 0 !important;
```

Comment:
If not important here, remove it.
And check and fix it elsewhere in the theme.

Reply:
solve this issue